### ISSUE TITLE
Crash when displaying the conversation list on Android 4.4

### ISSUE THREAD

**Issue by chendo** [intercom/intercom-cordova#115]
When attempting to view the conversation list on Android 4.4.2, the app crashes.

```
E/AndroidRuntime( 7972): java.lang.RuntimeException: Unable to start activity ComponentInfo{au.com.drmedev/io.intercom.android.sdk.activities.IntercomMessengerActivity}: java.lang.NullPointerException
E/AndroidRuntime( 7972):       	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2412)
E/AndroidRuntime( 7972):       	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2470)
E/AndroidRuntime( 7972):       	at android.app.ActivityThread.access$900(ActivityThread.java:174)
E/AndroidRuntime( 7972):       	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1307)
E/AndroidRuntime( 7972):       	at android.os.Handler.dispatchMessage(Handler.java:102)
E/AndroidRuntime( 7972):       	at android.os.Looper.loop(Looper.java:146)
E/AndroidRuntime( 7972):       	at android.app.ActivityThread.main(ActivityThread.java:5593)
E/AndroidRuntime( 7972):       	at java.lang.reflect.Method.invokeNative(Native Method)
E/AndroidRuntime( 7972):       	at java.lang.reflect.Method.invoke(Method.java:515)
E/AndroidRuntime( 7972):       	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1283)
E/AndroidRuntime( 7972):       	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1099)
E/AndroidRuntime( 7972):       	at dalvik.system.NativeStart.main(Native Method)
E/AndroidRuntime( 7972): Caused by: java.lang.NullPointerException
E/AndroidRuntime( 7972):       	at io.intercom.android.sdk.inbox.InboxFragment.displayEmptyView(InboxFragment.java:402)
E/AndroidRuntime( 7972):       	at io.intercom.android.sdk.inbox.InboxFragment.onStateChange(InboxFragment.java:369)
E/AndroidRuntime( 7972):       	at io.intercom.android.sdk.inbox.InboxFragment.onStateChange(InboxFragment.java:57)
E/AndroidRuntime( 7972):       	at io.intercom.android.sdk.utilities.ChangeSubscriber.onStateChange(ChangeSubscriber.java:33)
E/AndroidRuntime( 7972):       	at io.intercom.android.sdk.store.Store.subscribe(Store.java:87)
E/AndroidRuntime( 7972):       	at io.intercom.android.sdk.store.Store.subscribeToChanges(Store.java:96)
E/AndroidRuntime( 7972):       	at io.intercom.android.sdk.inbox.InboxFragment.onCreate(InboxFragment.java:124)
E/AndroidRuntime( 7972):       	at android.support.v4.app.Fragment.performCreate(Fragment.java:2075)
E/AndroidRuntime( 7972):       	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1060)
E/AndroidRuntime( 7972):       	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1295)
E/AndroidRuntime( 7972):       	at android.support.v4.app.BackStackRecord.run(BackStackRecord.java:801)
E/AndroidRuntime( 7972):       	at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1682)
E/AndroidRuntime( 7972):       	at android.support.v4.app.FragmentController.execPendingActions(FragmentController.java:388)
E/AndroidRuntime( 7972):       	at android.support.v4.app.FragmentActivity.onStart(FragmentActivity.java:607)
E/AndroidRuntime( 7972):       	at android.support.v7.app.AppCompatActivity.onStart(AppCompatActivity.java:181)
E/AndroidRuntime( 7972):       	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1177)
E/AndroidRuntime( 7972):       	at android.app.Activity.performStart(Activity.java:5468)
E/AndroidRuntime( 7972):       	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2385)
E/AndroidRuntime( 7972):       	... 11 more
```

## Version info
  - Intercom for Cordova/PhoneGap version: 3.0.16
  - iOS/Android OS version: 4.4.2
  - iOS/Android Device: Samsung GT-I9195

## Expected behavior

User sees conversation list.

## Actual behavior

App crashes on 4.4.2. Same build works fine on 7.0 and 6.0.1.

## Steps to reproduce
 1. Activate Intercom.
 2. Navigate to conversation list.

**Comment by ronocod** [comment]
Thanks for reporting this, we're working on a fix now.

**Comment by ronocod** [comment]
This is fixed in 3.0.17.
